lib/katalyst/tables/frontend.rb

```diff
@@ -16,7 +21,31 @@ def table_with(collection:, **options, &block)
 
         html_options = html_options_for_table_with(**options)
 
-        Builder.new(self, collection, table_options, html_options).build(&block)
+        builder = options.fetch(:builder) { default_table_builder_class }
+        builder.new(self, collection, table_options, html_options).build(&block)
+      end
+
+      def table_header_row(table, builder, &block)
+        builder.new(table).build(&block)
+      end
+
+      def table_header_cell(table, method, builder, **options)
```

Comment:
**Question:** I'm assuming (as the others have) you're passing `&block`'s through to provide additional flexibility - however is there a reason for avoiding that on this particular header cell function?

Reply:
Yes – it's not used in the header because it's usually not useful there. There's a later commit that adds it I think.